app/src/androidTest/java/com/sdp/movemeet/view/activity/ListActivityTest.java

```diff
@@ -0,0 +1,54 @@
+package com.sdp.movemeet.view.activity;
+
+import androidx.annotation.NonNull;
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.FirebaseDatabase;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.CountDownLatch;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class ListActivityTest {
+
+    private FirebaseAuth fAuth;
+    private FirebaseDatabase database;
+
+    @Test
+    public void listOfActivityTest() {
```

Comment:
What's the purpose of this test ? It looks like it does nothing

Reply:
Yes, this test has to be implemented by Roxane, but is apparently not finished yet! This is a file to test her new Android activity ActivityListActivity.